### PR TITLE
Rerender

### PR DESCRIPTION
**Pull request by ocefpaf**
Should add the Python matrix.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,26 @@
...
   build:
     - python
-    - setuptools
+    - pip
   run:
     - python
     - gazar
     - netcdf4
-    - pynio     # [not py3k and not win]
+    - pynio  # [not py3k and not win]
```
Comment (ocefpaf) [inline recipe/meta.yaml:23]: @snowman2 is this an optional dependency?  If we can remove that we can make this recipe `noarch` and build it much faster.

Comment (ocefpaf) [reply recipe/meta.yaml:23]: According to the `setup.py` upstream only  `gazar` and `wrf-python` are mandatory dependencies.

**Comment by dharhas** [comment]
@ocefpaf you made a bunch of changes in the rerender step, is this a conda-smithy function? I need to do something similar for the downstream gsshapy feedstock https://github.com/conda-forge/gsshapy-feedstock, so if you can point me to documentation...

File: recipe/meta.yaml (hunk 1 above)
Comment (dharhas) [reply recipe/meta.yaml:23]: is noarch fully supported now? i.e. should I be using that for pure python recipes?

Comment (snowman2) [reply recipe/meta.yaml:23]: If I remember correctly, that should be fine.

Comment (ocefpaf) [reply recipe/meta.yaml:23]: > is noarch fully supported now?

The only issue with `noarch` is when using the package with `conda-constructor`. If you are not using constructor and `pynio` dependency can be removed I highly recommend making this `noarch`.

**Comment by ocefpaf** [comment]
> @ocefpaf you made a bunch of changes in the rerender step, is this a conda-smithy function? I need to do something similar for the downstream gsshapy feedstock https://github.com/conda-forge/gsshapy-feedstock, so if you can point me to documentation...

All the changes in the first commit are `conda-smithy`. There two way to do that.

1. re-render manually: https://conda-forge.org/docs/conda_smithy.html
2. send a PR and use the webservices: https://conda-forge.org/docs/webservice.html#conda-forge-admin-please-rerender

**Comment by dharhas** [comment]
@ocefpaf nice. I like the webservice.

**Pull request merged**